packages/terra-dialog-modal/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "terra-dialog-modal",
+  "main": "lib/DialogModal.js",
+  "version": "0.0.0",
+  "description": "The Dialog Modal would allow for dynamic height modals. Dialog modal would be a limited use case, as dynamic heights breaks with more complicated DOM structures. If content is too complicated, the terra-modal-manager should be used. The components is placed at an 8000 z-index. The dialog supports release and request focus props similar to terra-popup and terra-date-picker, so it can be presented from another modal with focus.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-framework.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Framework",
+    "terra-dialog-modal",
+    "DialogModal",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-framework/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-framework#readme",
+  "peerDependencies": {
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "terra-base": "^3.1.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "prop-types": "^15.5.8",
+    "terra-abstract-modal": "^1.1.0",
+    "terra-action-footer": "^1.5.0",
+    "terra-action-header": "^1.2.0",
+    "terra-app-delegate": "^2.2.0",
+    "terra-base": "^3.1.0",
```

Comment:
Change terra-base to peer only dependency.

Reply:
This is a change discussed on https://github.com/cerner/terra-framework/issues/200 if you are curious